<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rs::QtV8GUI</class>
 <widget class="QWidget" name="rs::QtV8GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtV8GUI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTextEdit" name="textEdit_code"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_run">
       <property name="text">
        <string>运行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_clean">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_debug">
       <property name="text">
        <string>调试</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTextBrowser" name="textBrowser_log"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
